script/bootstrap

```diff
@@ -58,6 +58,9 @@ install_packages_apt()
 
     # add mdv for local size report
     python3 -m pip install mdv || echo 'Failed to install markdown render. Install it manually if you need.'
+
+    # add shfmt for shell pretty
+    command -v shfmt || brew install shfmt || (sudo apt-get install --no-install-recommends -y snapd && sudo snap install shfmt)
```

Comment:
Does attempting to use `brew install` make sense in `install_packages_apt()`?

Reply:
Yes, snap doesn't support home dir not being at /home. Linuxbrew works perfect. So try brew first.

Reply:
Makes sense. Can we add a comment since having `brew install` inside `install_packages_apt()` is not intuitive?

Reply:
added.